data-management/data-interface.js

```diff
@@ -83,10 +85,8 @@ async function getIcdcStudyIds() {
       body: body,
     });
     const data = await response.json();
-    const studyIds = data.data.studiesByProgram.map(
-      (obj) => obj.clinical_study_designation
-    );
-    return studyIds;
+    const studyData = data.data.studiesByProgram;
```

Comment:
if data.data is null then the error will be caught by the catch block and will be reported as BENTO_BACKEND_NOT_CONNECTED. I think it would be better to either add more specific catch blocks or make this property chain null safe